alacritty_terminal/src/tty/windows/child.rs

```diff
@@ -4,14 +4,16 @@ use std::sync::atomic::{AtomicPtr, Ordering};
 
 use mio_extras::channel::{channel, Receiver, Sender};
 
-use winapi::shared::ntdef::{BOOLEAN, HANDLE, PVOID};
-use winapi::um::winbase::{RegisterWaitForSingleObject, UnregisterWait, INFINITE};
-use winapi::um::winnt::{WT_EXECUTEINWAITTHREAD, WT_EXECUTEONLYONCE};
+use windows_sys::Win32::Foundation::{BOOLEAN, HANDLE};
+use windows_sys::Win32::System::Threading::{
+    RegisterWaitForSingleObject, UnregisterWait, WT_EXECUTEINWAITTHREAD, WT_EXECUTEONLYONCE,
+};
+use windows_sys::Win32::System::WindowsProgramming::INFINITE;
 
 use crate::tty::ChildEvent;
 
 /// WinAPI callback to run when child process exits.
-extern "system" fn child_exit_callback(ctx: PVOID, timed_out: BOOLEAN) {
+extern "system" fn child_exit_callback(ctx: *mut c_void, timed_out: BOOLEAN) {
```

Comment:
Are you sure that there's no special type for that on windows? What native apps use here instead?

Reply:
Yes, looking at the Windows-Sys documentation, this is how they defined this function. (https://docs.rs/windows-sys/latest/windows_sys/Win32/System/Threading/type.WAITORTIMERCALLBACK.html)